File: Spring-backend/Springbackend/src/main/java/com/dravicenne/backend/models/Horraire.java
package com.dravicenne.backend.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.io.Serializable;

@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
@Entity(name = "horraire")
public class Horraire implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String jour;
    private String heure_debut;
    private String heure_fin;
}

File: Angular-frontend/src/app/Models/horraire/horraire.ts
export class Horraire {
  id: number;
  jour: string;
  heure_debut: string;
  heure_fin: string;


  constructor(id: number, jour: string, heure_debut: string, heure_fin: string) {
    this.id = id;
    this.jour = jour;
    this.heure_debut = heure_debut;
    this.heure_fin = heure_fin;
  }
}
